src/tournamentcreator.cpp

```diff
@@ -3,6 +3,21 @@
 
 void MainWindow::loadTournamentCreatorPage()
 {
+    utilDebug("Load Tournament Creator Page");
+    int i = 0;
+    QString filePath;
+    while(true)
+    {
+        filePath = m_EventDirectory + m_EventDefaultFileName + std::to_string(i).c_str() + ".json";
```

Comment:
In this line change `std::to_string(i).c_str()` to be `std::to_string(i++).c_str()`